boa/contracts/vvm/vvm_contract.py

```diff
@@ -162,33 +162,31 @@ def internal(self):
         def internal():
             return None
 
-        result = self._compile_metadata_fn_info()
+        result = self._compile_function_metadata()
         for fn_name, meta in result.items():
             if meta["visibility"] == "internal":
                 function = VVMInternalFunction(meta, self)
                 setattr(internal, function.name, function)
         return internal
 
-    def _compile_metadata_fn_info(self) -> dict:
-        """Compiles the metadata for the contract"""
+    def _compile_function_metadata(self) -> dict:
+        """Compiles the contract and returns the function metadata"""
+
         # todo: move this to vvm?
```

Comment:
@charles-cooper if you agree I can make a PR

Reply:
yes, although it's version specific. i think in recent versions, it should come with `combined_json` output

Reply:
another reason to move it to the vvm :nerd_face: